src/civ2.js

```diff
@@ -52,183 +65,148 @@ module.exports = function(robot) {
         const tagTxt = tag ? `tag ${tag}` : 'default tag';
         msg.reply(`The deployment of ${tagTxt} to ${target.trigger} is scheduled.`);
       } catch (ex) {
-        console.error(ex);
+        error(ex);
```

Comment:
You should call the function `logError`, it's confusing like this since it's not a verb. And having all log functions sharing a same prefix helps readability.